### ISSUE TITLE
need prometheus_ex 3.0.2 for elixir 1.7

### ISSUE THREAD

**Issue by whenceforth** [deadtrickster/prometheus-phoenix#5]
prometheus_ex doesn't compile under elixir 1.7 due to 
https://github.com/deadtrickster/prometheus.ex/issues/18

That's been fixed as of version 3.0.2 of prometheus_ex, so an upgrade of the version here would probably solve it for prometheus-phoenix.

**Comment by deadtrickster** [comment]
Yes, but `~>` operator used here (`~> 3.0`) so should download the latest 3.x.x version. I could remove 'ors' in the mix.exs but this will force everyone to update to 3.x.x. Need to think about it more. In meantime please use override.

**Comment by whenceforth** [comment]
Unfortunately, that doesn't work for me because I'm also using prometheus_process_collector.
prometheus_ex 3.0.2 requires prometheus 4.0 but prometheus_process_collector (version 1.3.1) requires prometheus `~> 3.4`
